src/main.ts

```diff
@@ -180,6 +193,12 @@ async function run(): Promise<void> {
       }
     } catch (err) {
       core.warning(`Failed to get Tailscale status: ${err}`);
+      if (runnerOS === runnerMacOS) {
+        core.setFailed(
+          `❌ Tailscale status is required in order to configure macOS`
+        );
+        process.exit(2);
```

Comment:
Is there a reason we exit with status code 2 here specifically?

Reply:
I just wanted something different so that one can distinguish this particular condition if need be.